Authentication/Login/View/LoginConfiguration.swift

```diff
@@ -13,7 +13,7 @@ import Foundation
 */
 public protocol LoginControllerTransitionDelegate {
```

Comment:
So do I add that method `userDidLogIn`? The only thing I don't like with that, is that the `SessionService`'s events signal becomes somewhat useless since we will only use the `LogOut` event to redirect to login, which is great, but I think we shouldn't demand many events from the service if we will only use one.